frontend/src/metabase/modes/components/drill/ObjectDetailDrill.jsx

```diff
@@ -1,12 +1,10 @@
-/* eslint-disable react/prop-types */
-import { isFK, isPK } from "metabase/lib/schema_metadata";
 import { t } from "ttag";
+import { isFK, isPK } from "metabase/lib/schema_metadata";
 
 export default ({ question, clicked }) => {
   if (
-    !clicked ||
-    !clicked.column ||
-    clicked.value === undefined ||
+    !clicked?.column ||
+    clicked?.value === undefined ||
     !(isFK(clicked.column) || isPK(clicked.column))
```

Comment:
Unrelated, maybe later we can rename FK and PK to their more complete names.

Looking at this diff I have no idea what these are :)